.github/workflows/build-test-publish.yml

```diff
@@ -17,6 +17,7 @@ env:
   # Taken from https://github.com/kubernetes-sigs/kind/releases/tag/v0.18.0
   # The image here should be listed under 'Images built for this release' for the version of kind in go.mod
   KIND_NODE_IMAGE: "kindest/node:v1.26.3@sha256:61b92f38dff6ccc29969e7aa154d34e38b89443af1a2c14e6cfbd2df6419c66f"
+  KIND_119_NODE_IMAGE: "kindest/node:v1.19.16@sha256:81f552397c1e6c1f293f967ecb1344d8857613fb978f963c30e907c32f598467"
```

Comment:
Would it make sense to call this `KIND_OLDEST_NODE_IMAGE` or something similar? Then, if we decide to make a change to support only K8s versions from some newer release, we won't have to rename this.

Reply:
Updated